jupyterlab_table/utils.py

```diff
@@ -72,10 +73,14 @@ def sanitize_dataframe(df):
     return df
 
 
-def prepare_data(data=None):
-    """Prepare Plotly data from Pandas DataFrame."""
+def prepare_data(data=None, schema=None):
+    """Prepare JSONTable data from Pandas DataFrame."""
     
-    if isinstance(data, list):
-        return data
-    data = sanitize_dataframe(data)
-    return data.to_dict(orient='records')
+    if isinstance(data, pd.DataFrame):
+        data = sanitize_dataframe(data)
```

Comment:
This `sanitize_dataframe` code comes from the original jupyterlab_vega which used `to_dict(orient='records')`. I think that `to_json` should take care of our sanitization needs?